packages/components/src/resizable-box/index.js

```diff
@@ -23,7 +23,8 @@ function ResizableBox( { className, ...props } ) {
 		<Resizable
 			className={ classnames(
 				'components-resizable-box__container',
-				className,
+				showHandle && 'is-show-handle',
```

Comment:
Did you consider an alternative name for `is-show-handle`? It doesn't read great. `shows-handle` would be better but it probably doesn't follow the naming convention for the class names. Maybe `has-resize-handle` based on the code comment?

Reply:
Ah yes! `has-resize-handle` reads better. It's a struggle sometimes with these names 😂